api/azure/sdk_type_definition.rb

```diff
@@ -99,6 +99,11 @@ class ComplexArrayObject < ComplexObject
       class StringMapObject < SDKTypeDefinition
       end
 
+      class Integer32ArrayObject < SDKTypeDefinition
```

Comment:
*minor* Can we please move these lines up, so all `array` related types are bundled together in this file.

Reply:
OK